modules/social_features/social_event/social_event.install

```diff
@@ -2295,3 +2295,32 @@ function social_event_update_11903(): void {
     \Drupal::logger('social_event')->error($e->getMessage());
   }
 }
+
+/**
+ * Update the views that are associated with enrolling in an event.
+ *
+ * To prevent duplication of users.
+ */
+function social_event_update_11904() {
```

Comment:
@Kovalskiy266 Please change it also in the original config files, not only in the update hook.

Reply:
Done